src/main/java/org/folio/rest/transaction/AllOrNothingHandler.java

```diff
@@ -376,7 +376,8 @@ protected String getValues(List<JsonObject> entities) {
 
   private Future<Void> checkEncumbranceRestrictions(Transaction transaction, Budget budget) {
     Promise<Void> promise = Promise.promise();
-    if (transaction.getTransactionType() == Transaction.TransactionType.ENCUMBRANCE) {
+    if (transaction.getTransactionType() == Transaction.TransactionType.ENCUMBRANCE
```

Comment:
I think Credits can be applied only to `Active` budgets as well. @craigmcnally  is that correct?

Reply:
That's correct.  Apologies for the delay